javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,79 @@
+console.log("hello");
+//building a voice assistant.
+//function called getReply(command).
+let personName = [];
```

Comment:
I would say a username can be only one. 
```suggestion
let personName = "";
```

Reply:
Yes you are right I will make  changes on this. thanks